src/Observable.ts

```diff
@@ -146,6 +146,7 @@ export class Observable<T> implements CoreOperators<T>  {
                                removeHandler: (handler: Function) => void,
                                selector?: (...args: Array<any>) => T) => Observable<T>;
   static fromPromise: <T>(promise: Promise<T>, scheduler?: Scheduler) => Observable<T>;
+  static fromCallback: <T>(callbackFunc: Function, ctx: Object, selector: Function, scheduler: Scheduler) => Function;
```

Comment:
parameter other than `callbackFunc` is allowed to be optional in signature of `create`, 

`create<T>(callbackFunc: Function, ctx: Object = undefined, selector: Function = undefined, scheduler: Scheduler = immediate)`

where this signature doesn't.